api/services/Flairs.js

```diff
@@ -116,26 +116,24 @@ exports.getFlairTextForSVEx = function (user) {
   }
   return flairText;
 };
-exports.canUserApply = function (user, applicationFlair, allflairs) {
-  if (typeof applicationFlair === 'string') {
-    applicationFlair = exports.getFlair(applicationFlair, allflairs);
-  }
-  if (!user || !user.references || !applicationFlair || exports.userHasFlair(user, applicationFlair) || exports.applied(user, applicationFlair)) {
+exports.canUserApply = function (refs, applicationFlair, currentFlairs) {
```

Comment:
I changed this so that it can be called properly from both the client and the server.